infra/chart/templates/deployment.yaml

```diff
@@ -111,7 +111,7 @@ spec:
         - name: LAYOUT
           value: "BaseLayout"
         - name: SWAGGER_JSON
-          value: "/app/swagger.yaml"
+          value: "/app/openapi.yaml"
         - name: PORT
           value: "{{ .Values.swaggerUi.service.internalPort }}"
         - name: API_HOST
```

Comment:
Is anything about this configuration expected to change with OpenAPI V3, since we are using the server URL instead of host and basepath?

Reply:
Likely, yes. As well as the deployment chart and its `initContainer` script. Will address this

Reply:
Done